CRM/Contact/Import/Form/MapField.php

```diff
@@ -546,7 +548,7 @@ public function submit($params, $mapperKeys) {
           $locations[$i] = $locationTypes[$selOne];
           $parserParameters['mapperLocType'][$i] = $selOne;
           if ($selTwo && is_numeric($selTwo)) {
-            if ($fldName === 'phone') {
+            if ($fldName === 'phone' || $fldName === 'phone') {
```

Comment:
Based on the rest this should be `phone_ext`

Reply:
Fail, thx for the revision 
https://github.com/civicrm/civicrm-core/pull/20995/commits/3aca1423bad7c7dd9d2e0f9687b85823a2b64bb5